lighthouse-core/audits/is-on-https.js

```diff
@@ -30,6 +30,22 @@ const UIStrings = {
     }`,
   /** Label for a column in a data table; entries in the column will be the URLs of insecure (non-HTTPS) network requests. */
   columnInsecureURL: 'Insecure URL',
+  /** Label for a column in a data table; entries in the column will be how the browser handled insecure (non-HTTPS) network requests. */
+  columnResolution: 'Resolution',
+  /** Value for the resolution column in a data table; denotes that the insecure URL was allowed by the browser. */
+  allowed: 'Allowed',
+  /** Value for the resolution column in a data table; denotes that the insecure URL was blocked by the browser. */
+  blocked: 'Blocked',
+  /** Value for the resolution column in a data table; denotes that the insecure URL may be blocked by the browser in the future. */
+  warning: 'Warned',
+  /** Value for the resolution column in a data table; denotes that the insecure URL was upgraded to a secure request by the browser. */
+  upgraded: 'Automatically Upgraded',
+};
+
+const resolutionToString = {
+  MixedContentAutomaticallyUpgraded: UIStrings.upgraded,
+  MixedContentBlocked: UIStrings.blocked,
+  MixedContentWarning: UIStrings.warning,
```

Comment:
these have to be run through `str_` (here or down in `audit()`) to get localized

Reply:
oh my. and the tests don't fail..

Reply:
> oh my. and the tests don't fail..

yeah, and I can't think of a great way to catch something like that :/

I suggested in #10630 that we could tighten some types to require i18n references and never allow strings, but it seems like it would work for things like `displayValue`. It would probably be way too restrictive to never allow non-localized strings in table items...

Other than that, search for every string in test LHRs and see if they're a value in `en-US.json`? Probably only like 20% flakey :)